google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/AppEngineDeploy.java

```diff
@@ -198,37 +145,29 @@ public void exit(int exitCode) {
 
           if (deployOutput == null
               || deployOutput.getService() == null || deployOutput.getVersion() == null) {
-            consoleLogLn(
-                GctBundle.message("appengine.deployment.version.extract.failure") + "\n");
+            loggingHandler.print(
```

Comment:
I've got a terrible memory, but didn't we say messages to logs should not be localized, only messages in the UI elements?